.gitignore

```diff
@@ -278,9 +278,9 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
-=======
->>>>>>> origin/main
 
 # Pycharm IDE settings
 .idea/
 .Rproj.user
+Data_Collection/.Rhistory
+Data_Collection/.RData
```

Comment:
it is a very small point but ... it is a common approach to add an empty line at the end of the file.
you can find some explanations here: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

you can configure your IDE to do it automatically